test/meta-helpers.test.ts

```
@@ -0,0 +1,49 @@
+        location: "Plaça Major",
+        city: { id: 1, name: "Vic", slug: "vic", postalCode: "08500" },
+        region: { id: 2, name: "Osona", slug: "osona" },
+        startDate: "2025-06-20",
+        endDate: "2025-06-20",
+        slug: "concert-de-prova-1",
+        categories: [{ id: 10, name: "Música", slug: "musica" }],
+        visits: 0,
+        imageUrl: "https://example.com/image.jpg",
+        relatedEvents: [],
+        weather: null,
+      } as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)